Dockerfile

```diff
@@ -1,35 +1,16 @@
-FROM python:3.7.0
+FROM jupyter/base-notebook:61d8aaedaeaf
 MAINTAINER "chestercheng <chester7864@gmail.com>"
 
-# Install requirements
-RUN pip install matplotlib==3.0.0
-RUN pip install numpy==1.15.3
-RUN pip install ipykernel==5.1.0
-RUN pip install ipython==7.0.1
-RUN pip install jupyter==1.0.0
-RUN pip install jupyter-contrib-nbextensions==0.5.0
+ADD * /home/$NB_USER/work/
```

Comment:
@chestercheng I think we don't need $NB_USER too. We will use jovyan for people in all docker container :)

Reply:
@hlb ` $NB_USER ` is define in [jupyter/base-notebook Dockerfile](
https://github.com/jupyter/docker-stacks/blob/master/base-notebook/Dockerfile).
> Line 10.
> ```
> ARG NB_USER="jovyan"
> ...
> ```

It needs root permission when installing jupyter notebook extensions.
` $NB_USER ` at the last is make sure to switch back to non-root user (jovyan).

Reply:
oh great :D